test/spec.js

```diff
@@ -24,6 +24,23 @@
 
 	var animationFrame = 1000/60;
 
+	if(window._) {
+		_.debounce = function debounce(func, wait, options) {
```

Comment:
are we trying to override underscore's implementation with this?
Will this break anything else on the page using underscore?